README.rst

```diff
@@ -84,6 +84,14 @@ Alternatively, the specifications can be installed through conda
 (C, C++, Fortran, Python) or Maven (Java).
 See the links above for details.
 
+The default branch of this repository
+reflects the current state of development for the BMI.
+When implementing a BMI,
+please use the latest release listed in the right sidebar;
```

Comment:
how about linking `latest release` to https://github.com/csdms/bmi/releases ?

Reply:
Done! Thanks, @alee!